helm/tests/ensure.py

```diff
@@ -430,7 +433,7 @@ def awscluster(kubernetes_cluster):
     LOGGER.info(f"AWSCluster {cluster_name} applied")
 
     raw = kubernetes_cluster.kubectl(
-        f"get awscluster {cluster_name}", output="yaml")
+        f"get awsclusters.v1alpha3.infrastructure.cluster.x-k8s.io {cluster_name}", output="yaml")
```

Comment:
We are using v1alpha4 as the stored version on etcd, so we need to be explicit about which api version we want so conversion webhooks are called.